packages/here-wallet/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@near-wallet-selector/here-wallet",
+  "version": "1.0.0"
```

Comment:
Can you update the version and set it to the same version as the other packages?

Also for packages that depend on `near-api-js` we need to provide it as a peerDependency the same way we have done for `@near-wallet-selector/my-near-wallet` and other packages see here:

https://github.com/near/wallet-selector/blob/main/packages/my-near-wallet/package.json#L4-L6